### DISCUSSION TITLE
links/

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#118] [Comments]
欢迎互加友链，请在本issue中回复。

添加本博客链接： 

### 胡涂说： https://hutusi.com/ 
任抛星汉归园圃，留取乾坤盛酒囊。

**Comment by top** [comment]
Ray: https://ie9.org
北美创业者

**Comment by hutusi** [comment]
> Tom: https://ie9.org
> 北美创业者的日常

@top 已加，谢谢~

**Comment by oeddyo** [comment]
ElasticSearch教程：https://kalasearch.cn/blog/elasticsearch-tutorial/

最全面的Elastic Search教程

**Comment by hutusi** [comment]
> ElasticSearch教程：https://kalasearch.cn/blog/elasticsearch-tutorial/
> 
> 最全面的Elastic Search教程

希望友链是博客链接，你看能否修改下？谢谢~~

**Comment by oeddyo** [comment]
Hi @hutusi 
这是就是我的技术博客，kalasearch.cn/blog

**Comment by hutusi** [comment]
> Hi @hutusi
> 这是就是我的技术博客，kalasearch.cn/blog

已加。

**Comment by petite-dd** [comment]
孤斗：http://d-d.design
做自己，不随波逐流，不妥协
我已链

**Comment by hutusi** [comment]
> 孤斗：http://d-d.design
> 做自己，不随波逐流，不妥协
> 我已链

谢谢，已加 ~~

**Comment by xdmjun** [comment]
薛定喵君的博客：http://xuedingmiao.com

**Comment by hutusi** [comment]
@xuedingmiaojun 已加，谢谢~~

**Comment by tianheg** [comment]
一大加贝 https://blog.yidajiabei.xyz/

学习技术，热爱生活

**Reply by tianheg** [reply]
Hello, 博客地址更新了 https://tianheg.co

**Reply by hutusi** [reply]
已更新~

**Comment by hutusi** [comment]
@tianheg 已加。大三就这么厉害，加油 👍

**Comment by surmon-china** [comment]
Surmon https://surmon.me

**Comment by hutusi** [comment]
@surmon-china 已加👍

**Comment by 17678319606** [comment]
我可真讨厌要用特殊网络的……
好了，交换：
李雪含
https://dajiayouxuan.com/
每天分享几篇值得看的商业，创业或者投资的文章

**Reply by hutusi** [reply]
啊，你是指哪个需要特殊网络？访问我的博客需要用特殊网络吗？

**Comment by 17678319606** [comment]
发评论需要，GitHub常常出现访问不了的情况

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: John Hu ***@***.***>
发送时间: Monday, April 4, 2022 5:14:48 PM
收件人: hutusi/hutusi.github.com ***@***.***>
抄送: 李雪含 ***@***.***>; Comment ***@***.***>
主题: Re: [hutusi/hutusi.github.com] links/ (Discussion #118)


啊，你是指哪个需要特殊网络？访问我的博客需要用特殊网络吗？

―

**Reply by hutusi** [reply]
你这是用iPhone直接在邮件中回复的吗？

友链已加。

**Comment by libasten** [comment]
飞刀博客 https://www.feidaoboke.com/
独立的个人博客，涵盖读书、随笔、足球、信息技术和新闻评论。

**Reply by hutusi** [reply]
已加友链。另外，麻烦有处的更正：请将我的博客名改成「胡涂说」，不是「糊涂说」哈，谢谢~~

**Comment by 17678319606** [comment]
对着的啊

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: John Hu ***@***.***>
发送时间: Tuesday, April 5, 2022 8:35:08 AM
收件人: hutusi/hutusi.github.com ***@***.***>
抄送: 李雪含 ***@***.***>; Comment ***@***.***>
主题: Re: [hutusi/hutusi.github.com] links/ (Discussion #118)


已加友链。另外，麻烦有处的更正：请将我的博客名改成「胡涂说」，不是「糊涂说」哈，谢谢~~

―

**Comment by bearfm** [comment]
贵站友链已经放到我的页面里了，我的小破站信息如下：
URL：https://Y.cool
SITE：歪酷博客
（暂定叫这个吧，欢迎大家给我的博客起名）

**Reply by hutusi** [reply]
已加友链🤝。你的博客feeds能订阅吗？我在 [feeds.pub](https://feeds.pub/hutusi)订阅贵站时报错。

**Comment by semyin** [comment]
semyin: https://semyin.com

**Reply by semyin** [reply]
RSS: https://semyin.com/atom.xml

**Reply by hutusi** [reply]
已加，已订阅。🤝

**Comment by maoruibin** [comment]
名称：咕咚同学
描述：咕咚的个人博客
地址：http://gudong.site
头像：http://gudong.site/static/img/logo.jpg

**Reply by hutusi** [reply]
博客有问题？好像是后台报错

**Reply by hutusi** [reply]
已加🤝

**Comment by iamgodot** [comment]
「胡涂说」已加友链，我的博客：[Godot's Blog](https://iamgodot.com)，欢迎交流～

**Reply by hutusi** [reply]
谢谢，已加🤝

看了你的文章，写的很好，加油💪

**Reply by iamgodot** [reply]
非常感谢～向你学习，有机会多交流

**Reply by iamgodot** [reply]
不好意思，最近想要调整，麻烦把我的友链去掉吧，谢谢。

**Comment by BYGD** [comment]
地址：https://www.idonglei.com/
名称：冬雷博客
描述：三餐四季，从来都是一半生存一半诗，做到哪里算哪里咯

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by Keanu-42** [comment]
高质量博客，已添加友链，老哥不嫌弃的话也可以看看我的。
- 名称：Keanu-42's blog
- 链接：https://keanu-42.cn/
- 简介：小孩子不懂事，博客写着玩的 : )
----
关于RSS订阅的问题，我推荐老哥你使用 *[RSSHub](https://docs.rsshub.app/)* 这个浏览器插件，它可以自动嗅探网页上的RSS订阅，很方便。

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by chancelyg** [comment]
博客：https://www.chancel.me
个人简介：努力做一个没有标准答案的人

已添加贵站作为友链

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by LxnChan** [comment]
>名称：泠泫凝的异次元空间
地址：https://lxnchan.cn
头像：https://lxnchan.cn/assets/logo-v2.webp
订阅：https://lxnchan.cn/atom.xml
描述：先空着或由您来写均可

已将贵站链接添加至[本站友链页](https://lxnchan.cn/friends/)，欢迎前来访问！

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by yangchuansheng** [comment]
云原生实验室：https://icloudnative.io

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by besscroft** [comment]
名称：风与牧歌
地址：https://blog.besscroft.com
描述：旅行终将迎来终点。
Logo：https://blog.besscroft.com/avatar.png

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by huangwb8** [comment]
关注胡涂说好久了，等更新好辛苦呀 哈哈！新人博主一枚，建站2月余。自己也准备像前辈们一样，长期坚持更新，哪怕慢一些！小站在此，希望可以互链一波，我加您喽！
站点名称：浮云翩迁之间
站点地址：https://blognas.hwb0307.com
站点描述：百代繁华一朝都，谁非过客；千秋明月吹角寒，花是主人。
站点图标：https://blognas.hwb0307.com/logo.jpg

**Reply by hutusi** [reply]
谢谢，已加😀

**Comment by ghost** [comment]
关于胡博主好久啦！请求添加友链~
博客: 月神夜的树洞
网址: http://www.zounuo.cc/
描述：我的人生五象限。
RSS 订阅地址: http://www.zounuo.cc/index.php/feed/

**Reply by hutusi** [reply]
谢谢，已加😀

**Comment by ChestnutHeng** [comment]
博客: 子恒的博客
网址: https://chestnutheng.cn/
RSS 订阅地址: https://chestnutheng.cn/index.xml

**Reply by ChestnutHeng** [reply]
后台、互联网、读书和旅行

**Reply by hutusi** [reply]
谢谢，已加😀 加油💪

**Comment by ghost** [comment]
无名博客
https://wumingboke.xyz/

**Reply by hutusi** [reply]
没有看到博客内容和友链哈~

**Comment by rootdba** [comment]
博客: 牧人池塘
网址: https://dba.kim
说明：时光匆匆，记录是对思维的沉淀；
RSS 订阅地址: https://dba.kim/feed/

**Reply by hutusi** [reply]
谢谢，已加😀

**Comment by linlinzzo** [comment]
博客: 林林杂语
网址: https://www.xiaozonglin.cn/
RSS 订阅地址: https://www.xiaozonglin.cn/feed/
（请将RSS的刷新频率调低一点，谢谢）

**Reply by hutusi** [reply]
不好意思，有段时间没有处理博客了，已加链接。

**Comment by peter-liu** [comment]
一直阅读老师你的文章，斗胆申请一个友链。网站文章不多，不像你如此高产~

> 博客: 吐司面包
> 网址: https://toast.pub
> RSS 订阅地址: https://toast.pub/totoro/index.xml

首页有些其它东西，如果更偏好放博客地址的话，你看以下信息是否可以：

> 博客: Pangzi's Blog
> 网址: https://toast.pub/totoro/about
> RSS 订阅地址: https://toast.pub/totoro/index.xml


贵站链接我今天也已挂上：https://toast.pub/totoro/friends/

**Reply by hutusi** [reply]
有段时间没有处理博客了，已加链接：）

**Comment by hjroyal** [comment]
名称: 有意栽花花满枝
介绍：心中有光、点亮生活
地址：https://huangjia.site

**Reply by hutusi** [reply]
已加。新年快乐：）

**Comment by Oliyrs** [comment]
名称：夜路
地址：https://emmm.space/
简介：上士无争，下士好争
头像：https://img.beeble.eu.org/202301/android-chrome-512x512.png
RSS: https://emmm.space/atom.xml

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by zwz1437** [comment]
已链
博客: tanscp
网址: https://tanscp.com

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by hjroyal** [comment]
博主，我更换了域名，麻烦更新下

博客: 胡有意栽花花满枝
网址: https://blog.hjroyal.top

**Reply by hutusi** [reply]
已更新

**Comment by yzzgchina** [comment]
本站名称：南生余
本站链接：https://www.chenii.com/
本站描述：一个喜欢怀旧和复古的文人墨客
本站图标：https://www.chenii.com/favicon.ico

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by erikwang2013** [comment]
博客: 艾瑞可erik
网址: https://erik.xyz/
简介：一只PHP开发的程序猿，偶尔做做运维、Goland、Python、Java、摄影、画画、写作、顺便睡觉，反正整站都搞过。

**Reply by hutusi** [reply]
谢谢，已加~

**Comment by Zeriy4** [comment]
很喜欢你的网站，简洁而有内涵。已链。
网站主题：梦幻派的艺术花园
网站简介：以“梦幻风格”为主题的文学艺术网站。
网站地址：https://zeriy.art/
网站LOGO：https://s1.ax1x.com/2023/01/31/pS05Z5T.png

**Reply by hutusi** [reply]
谢谢，已加~

**Comment by sheepbox8646** [comment]
简介：一个高中生记录技术的个人博客
地址：[https://index.acbox.lol](https://index.acbox.lol)

**Reply by hutusi** [reply]
厉害厉害，高中生就这么专注技术了👍 已加友链

**Comment by Zeriy4** [comment]
&nbsp;好捏






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "hutusi/hutusi.github.com"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月31日(星期天) 上午8:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [hutusi/hutusi.github.com] links/ (Discussion #118)





 
不好意思，最近想要调整，麻烦把我的友链去掉吧，谢谢。

**Comment by soarhone** [comment]
https://dwith.com
我刚新建了一个友情链接页面(https://dwith.com/friend-link )，你的博客是我的第一个友情链接。

**Reply by soarhone** [reply]
哦，忘记写title了
博客标题: Johnny @ Life Clips
博客地址: https://dwith.com

**Reply by hutusi** [reply]
感谢：）已加友链~~

**Comment by yexia553** [comment]
厚脸皮来求添加友链~已经添加您的站点~

潘智祥
https://panzhixiang.cn
读书、生活、工作

**Reply by hutusi** [reply]
谢谢。已加友链。不过两个问题：
1、你将我站点的名字写错了，应该是“胡涂说”
2、建议转载文章标题标注为“转载”
谢谢~

**Reply by yexia553** [reply]
抱歉抱歉，两个地方都已修改~

**Reply by hutusi** [reply]
🤝

**Comment by erikwang2013** [comment]
友链已加。

艾瑞可erik
https://erik.xyz
一只PHP开发的程序猿，偶尔做做运维、Goland、Python、Java、摄影、画画、写作、顺便睡觉，反正整站都搞过。



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年04月01日 10:14 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [hutusi/hutusi.github.com] links/ (Discussion #118) |

抱歉抱歉，两个地方都已修改~

**Reply by hutusi** [reply]
之前已经加过啦~

**Comment by huhexian** [comment]
我的链接麻烦有空可以替换一下，现在是 yinji.org

**Reply by hutusi** [reply]
已改，我准备等空闲点把友链页面整理一下了~

**Comment by DeppWang** [comment]
https://depp.wang 一个爱折腾的程序员

**Reply by hutusi** [reply]
已加友链，谢谢~~

**Reply by DeppWang** [reply]
你的我也加上了。

**Reply by hutusi** [reply]
🤝

**Comment by Xutaotaotao** [comment]
https://taotaoxu.com/ 

技术备忘录

**Reply by hutusi** [reply]
已加，谢谢~

**Comment by nwsci** [comment]
博客: Code掘金 
网址: https://www.sodofast.cn/ 
RSS 订阅地址: https://www.sodofast.cn/feed/
简介：记录校医工作日常和程序员副业实践

**Reply by hutusi** [reply]
贵站打不开，抱歉

**Comment by twoconk** [comment]
申请友链，谢谢！
名称：呱牛笔记
链接：https://it3q.com/
简介：记录生活和学习的日常

**Comment by twoconk** [comment]
申请友链，谢谢！
名称：呱牛笔记
链接：https://it3q.com/
简介：记录生活和学习的日常

**Comment by twoconk** [comment]
申请友链，谢谢！
名称：呱牛笔记
链接：https://it3q.com/
简介：记录生活和学习的日常

**Reply by hutusi** [reply]
没看到链接

**Comment by rootdba** [comment]
牧人池塘 域名变更为 https://dbanote.com
之前的域名不能用了，烦请修改下·

**Reply by hutusi** [reply]
好的，最近比较忙，我找个时间处理下哈。

**Reply by hutusi** [reply]
已改

**Comment by SimonSUSU** [comment]
网站名称：SimonSu
网站地址：https://simonsu.cn/
网站标志： https://simonsu.cn/logo.png
网站简介：这是一个记录生活、工作点滴的个人博客。这里没有华丽的文字，没有像花一样美的图片，只有平静生活中所感、所见、所爱的记忆。

**Reply by SimonSUSU** [reply]
已检查过了，已链接了。
原因，我页面有CDN缓存和客户端浏览器缓存，时间设得有点久。。。
如果你没有看到，可以强制刷新一下（按 ShITFT+F5）
链接所在页： https://simonsu.cn/links/

**Comment by erikwang2013** [comment]
未发现你那网站友链



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年11月01日 09:57 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [hutusi/hutusi.github.com] links/ (Discussion #118) |

网站名称：SimonSu
网站地址：https://simonsu.cn/
网站标志： https://simonsu.cn/logo.png
网站简介：这是一个记录生活、工作点滴的个人博客。这里没有华丽的文字，没有像花一样美的图片，只有平静生活中所感、所见、所爱的记忆。

**Comment by SimonSUSU** [comment]
已检查过了，已链接了。
原因，我页面有CDN缓存和客户端浏览器缓存，时间设得有点久。。。
如果你没有看到，可以强制刷新一下（按 ShITFT+F5）
https://simonsu.cn/links/



在 Nov 4, 2024, 9:49 AM +0800，erik ***@***.***>，写道：

未发现你那网站友链



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年11月01日 09:57 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [hutusi/hutusi.github.com] links/ (Discussion #118) |

网站名称：SimonSu
网站地址：https://simonsu.cn/
网站标志： https://simonsu.cn/logo.png
网站简介：这是一个记录生活、工作点滴的个人博客。这里没有华丽的文字，没有像花一样美的图片，只有平静生活中所感、所见、所爱的记忆。

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***>

—
Reply to this email directly, view it on GitHub
<https://github.com/hutusi/hutusi.github.com/discussions/118#discussioncomment-11137708>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AG5L77KNSK6CBEI7FM3RH4TZ63HEHAVCNFSM5SOVYWY2U5DIOJSWCZC7NNSXTOSENFZWG5LTONUW63SDN5WW2ZLOOQ5TCMJRGM3TOMBY>
.
You are receiving this because you commented.Message ID: <hutusi/hutusi.
***@***.***>

**Reply by hutusi** [reply]
谢谢二位，已加~

**Comment by henices** [comment]
安全代码  https://www.usmacd.com
rss: https://www.usmacd.com/rss.xml

**Reply by hutusi** [reply]
谢谢，已添加~

**Comment by suuseer** [comment]
https://suus.me
在数字的海洋中，寻找属于自己的星辰。
已添加糊涂说

**Reply by hutusi** [reply]
已添加，不过你把我博客的名字写错了，应该是“胡涂说”。

另外，你的友链不太好找~

**Comment by hutusi** [comment]
2025/09/14 update: 
删除了一些网站失联或单方面删除友链的网站。
由于很多友链网站的博主可能没有长期维护的计划，很多已经失联，因此对于后续申请交换友链的网站，我会先简单浏览内容做判断是否交换友链，敬请理解。

**Comment by huhexian** [comment]
何德何能，居然被列入【对我有过帮助或影响的博客】。

**Reply by hutusi** [reply]
你太谦虚了。主要的两点：1. 几年前那会你组织的博客群也是我认真开始写博客的一个触发因素。2. 给予了推广帮助。

**Comment by MarsperL** [comment]
已添加友链，期待互链
博客: 云野阁
网址: https://yyg.js.cool/
RSS 订阅地址: https://yyg.js.cool/rss2.xml
简介： 闲云野鹤，八方逍遥

**Reply by hutusi** [reply]
不好意思，目前友链的条件之一是必须是独立域名。

**Comment by listener-He** [comment]
哈喽老胡 可以加个友链吗
博客: Honesty

网址: https://blog.hehouhui.cn

RSS 订阅地址: https://blog.hehouhui.cn/rss/feed.xml

简介： 从浩瀚无垠的知识中看到到一缕光。

**Reply by hutusi** [reply]
已加👌

**Comment by zx88cvb** [comment]
友链已添加，期待互换友链

博客: HaydenBi
网址: https://haydenbi.com
RSS 订阅地址: https://haydenbi.com/feed.xml
简介：Coffee & Indie Developer
头像: https://r2.haydenbi.com/about/about.png

**Reply by hutusi** [reply]
已加友链，写得不错，加油：）

**Reply by zx88cvb** [reply]
谢谢😄

**Comment by HiraethEcho** [comment]
Hi，希望加您的友链
blog: MEMEX
url: https://memex.keinmal.top
rss: https://memex.keinmal.top/index.xml
summary: 赛博空间隐蔽而开放的角落

**Reply by hutusi** [reply]
已加。为什么要用 .top 域名啊？这种域名貌似 SEO 极不好

**Comment by ws0132613-cmyk** [comment]
william smith
ws0132613@gmail.com
https://getmacos.com/gopanel/
thanks

**Reply by hutusi** [reply]
？广告吗？

**Comment by timoseven** [comment]
樱桃沟夹事:https://blog.timoq.com/
写给未来的自己

**Reply by hutusi** [reply]
网站的名称没有写对

**Comment by BYGD** [comment]
心痛，友链没了~~
